packages/wp-story-editor/src/index.js

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+// The __webpack_public_path__ assignment will be done after the imports.
+// That's why the public path assignment is in its own dedicated module and imported here at the very top.
+// See https://webpack.js.org/guides/public-path/#on-the-fly
+import './publicPath';
+
+/**
+ * External dependencies
+ */
+import Modal from 'react-modal';
```

Comment:
Since this import is only needed for `setAppElement()`, should we perhaps just re-export that function in `story-editor`?

This way, we don't have a direct dependency on `react-modal` in `wp-story-editor`

Reply:
It doesn't feel right to re-export a module as it is when it could be directly used in the wp-story-editor, are we doing this elsewhere too? What would be the benefit of having one less dependency in wp-story-editor?

Reply:
That's a good reason, makes sense!